app/main.py

```diff
@@ -34,9 +36,20 @@
 results: dict[UUID, str] = {}
 
 
+@lru_cache
+def get_settings() -> Settings:
+    """
+    Get environment variables from pydantic and cache them.
+    """
+
+    return Settings()
```

Comment:
Could you move this to `services.py`?
That's were we hold the other "services" for depedency injection (DI)